src/ui/public/vislib/visualizations/gauges/simple.js

```diff
@@ -104,7 +104,21 @@ export function SimpleGaugeProvider() {
       const tooltip = this.gaugeChart.tooltip;
       const isTooltip = this.gaugeChart.handler.visConfig.get('addTooltip');
       const yFieldFormatter = this.gaugeChart.handler.data.get('yAxisFormatter');
-      const fontSize = this.gaugeChart.handler.visConfig.get('gauge.style.fontSize');
+
+      let fontSize;
+      let deprecatedFontSize;
```

Comment:
Move deprecatedFontSize to inside try/catch block since it is not used any where else and than make it a `const`